### PR TITLE
python3Packages.minikerberos: init at 0.2.7

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Kerberos manipulation library in pure Python.

https://github.com/skelsec/minikerberos

Related to #81418

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
@fabaff please fix the merge conflict.

**Comment by fabaff** [comment]
> @fabaff please fix the merge conflict.

Done

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -3907,7 +3909,11 @@ in {
 
   minidb = callPackage ../development/python-modules/minidb { };
 
+<<<<<<< HEAD
```
Comment (SuperSandro2000) [inline pkgs/top-level/python-packages.nix:3912]: @fabaff You missed something here.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 108800` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
6 packages built:
</summary>
<ul>
<li>python37Packages.asysocks</li>
<li>python37Packages.minikerberos</li>
<li>python38Packages.asysocks</li>
<li>python38Packages.minikerberos</li>
<li>python39Packages.asysocks</li>
<li>python39Packages.minikerberos</li>
</ul>
</details>

File: pkgs/top-level/python-packages.nix (hunk 1 above)
Comment (fabaff) [reply pkgs/top-level/python-packages.nix:3912]: Yes, I did. Thanks @SuperSandro2000

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 108800` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>9 packages built:</summary>
  <ul>
    <li>python37Packages.asysocks</li>
    <li>python37Packages.minikerberos</li>
    <li>python37Packages.winsspi</li>
    <li>python38Packages.asysocks</li>
    <li>python38Packages.minikerberos</li>
    <li>python38Packages.winsspi</li>
    <li>python39Packages.asysocks</li>
    <li>python39Packages.minikerberos</li>
    <li>python39Packages.winsspi</li>
  </ul>
</details>

**Review by jonringer — APPROVED** [review]
LGTM

```
https://github.com/NixOS/nixpkgs/pull/108800

9 packages built:
python37Packages.asysocks python37Packages.minikerberos python37Packages.winsspi python38Packages.asysocks python38Packages.minikerberos python38Packages.winsspi python39Packages.asysocks python39Packages.minikerberos python39Packages.winsspi
```

thanks @SuperSandro2000

**Pull request merged**